### PR TITLE
make email field mandatory and format of email

### LINKED ISSUE #5247: Reporting a problem leads to blank page when not logged in
Reporting a problem leads to blank page when not logged in.

### Evidence / Screenshot (if possible)
image

### Relevant url?

https://openlibrary.org/contact

### Steps to Reproduce

1. Go to https://openlibrary.org/contact (while not logged in)
2. Report a problem
3. See how you'll be taken to a blank page

* Actual: Goes to blank page
* Expected: Goes to thank you page or another screen

### Details

- **Logged in (Y/N)?** N
- **Browser type/version?** FF
- **Operating system?** Mac OS
- **Environment (prod/dev/local)?** prod

### Related files

#3826

### PR DESCRIPTION
**Pull request by tonytamsf**
Closes #5247

This will make the email field required as well as formatting as a valid email address

### Technical
This only validates on the frontend.

### Testing
1) Leave email blank and submit, you will get a warning
2) Enter invalid email, you won't be able to submit, and you will see a warning about invalid format

### Screenshot
Screen Shot 2021-07-03 at 7 03 56 PM

### Stakeholders
@RayBB @jamesachamp

### CODE REVIEW

File: openlibrary/templates/support.html [hunk 1]
```diff
@@ -16,12 +16,12 @@ <h1>$_("How can we help?")</h1>
 
  <div class="formElement">
  <div class="label"><label for="name">$_("Your Name")</label></div>
- <div class="input"><input type="text" class="input" name="name" id="name"/></div>
+ <div class="input,required" ><input type="text" class="input" name="name" id="name"/></div>
```
Comment (jamesachamp) [inline openlibrary/templates/support.html:19]: Please revert this.  It is unrelated to the email validation.

As an aside, the list of classes in the `class` attribute must be separated by spaces, not commas.

**Review by jamesachamp — CHANGES_REQUESTED** [review]
Using the `required` class for validation is consistent with how other fields are being validated in this file.  Validation and submission worked properly when I tested this locally.  Once the requested change to the unrelated `div` classes has been made, this should be good to go.

File: openlibrary/templates/support.html (hunk 1 above)
Comment (tonytamsf) [reply openlibrary/templates/support.html]: @jamesachamp reverted. Thanks for the tip about `class` attribute

**Review by jamesachamp — APPROVED** [review]
Lgtm!  Thanks @tonytamsf!

**Comment by RayBB** [comment]
@tonytamsf thanks for this!

**Pull request merged**